examples/observability/eks-java-jmx/dev.tfvars

```diff
@@ -0,0 +1,2 @@
+grafana_api_key  = "<inset grafana api key>"
```

Comment:
we should put a note somewhere to guide users to store their API key in either SSM parameter or Secrets Manager and reference it through code so that they do not save their API key in plain text in their source code

Reply:
@bryantbiggs there is a note in the README.md on this.